Added in a way to automatically get users profile pictures and displa…

cmjchrisjones opened the pull request:
Caveat: This might not be the best was to do it, I think I've managed to not break anything but is a bit of a first for me with NodeJS/Express/SocketIO apps, so any feedback greatly appreciated.

There is a preview gif at the bottom showing what the changes in this commit achieves.

### Changes in this commit
- Added in a way to automatically get users profile pictures and displa…y names when they are joining the game.
- Also added the ability to run this via docker/docker compose

### Breaking Changes
- Removed hardcoded channel name from `chat-capture.js` and moved it into `.env.config`

### Minor Changes
- Updated `README.md with more instructions`
- Added example `.env` file

![Chakeyra](https://user-images.githubusercontent.com/3969086/101458835-4ba0af00-392f-11eb-929a-7b258ba1e6ab.gif)

clarkio commented:
Thanks so much for taking the time to add this Chris and what a great idea! I'll review and ask any questions (if any) soon. If all looks good I'll just merge.

Thanks again!

@all-contributors please add @cmjchrisjones for code and ideas

clarkio commented:
@all-contributors please add @cmjchrisjones for code and ideas

cmjchrisjones commented:
Hey @clarkio I think I've fixed those bugs that were found when you reviewed this last time.

Also, I've moved the player list area, and prevented names overlapping, thoughts?

![image](https://user-images.githubusercontent.com/3969086/102730740-98bc5200-432d-11eb-8225-d7d969234e0c.png)

clarkio reviewed (approved):
We made a few changes on stream, refactoring a few things but everything is looking good and working as expected. Feel free to take a look to see what changed and let me know what you think but we're good to go otherwise and I'll merge everything.

Thanks again for taking the time to make this contribution!
